Docker/README.md

```diff
@@ -709,12 +746,12 @@ REPOSITORY      TAG        IMAGE ID       CREATED             VIRTUAL SIZE
 owncloud        latest     de9663de44b2   7 minutes ago       568.5 MB
 ```
 
-Now what would be great, would be to save our content from one run to another in order to find it again right ? And yes, you can ;-) For that, you need to attach a local directory of your host to your container, and point the setup of your owncloud to that directory instead of the one under `/var/www/html/owncloud`.
+At the moment, any content you create within the container will be lost when you it stop the container. Wouldn't be greate if your changes persisted from one run to another? Yes, you can ;-) For that, you need to attach a local directory of your host to your container, and point the setup of your owncloud to that directory instead of the one under `/var/www/html/owncloud`.
```

Comment:
It may sound strange in english, but I prefer to not be too direct saying "will be lost" but to generate into the person practicing a question to him/herself reading my comment.

I'm not understanding the "when you it stop" not am sure of the greate ?